src/common/thrift/ThriftClientManager.h

```diff
@@ -35,7 +35,7 @@ class ThriftClientManager final {
         std::shared_ptr<ClientType>                 // Async thrift client
     >;
 
-    folly::ThreadLocal<ClientMap> clientMap_;
+    ClientMap                     clientMap_;
```

Comment:
Why not use static clientMap_ directly?

Reply:
Whether to use static here is the same, because we use the ThriftClientManager object as thread_local.

Reply:
I mean why not use “static folly::ThreadLocal clientMap”

Reply:
If we used "static folly::ThreadLocal clientMap", all used clients (meta/storage) will share a clientMap. For example, adminclient and storageclient will likely use the same StorageServiceAsyncClient object. Is this necessary?

Reply:
I think it's ok for connections.   The same thread share the same connection.  wdyt?